apps/official/components/pages/Faq.tsx

```diff
@@ -258,6 +258,9 @@ export const Faq: React.FC = () => {
             Kontaktzahlen (Heterogenität). Der Kontakt-Index zeigt nun sowohl die Gesamtzahl der Kontakte als auch die
             Heterogenität an. Wir konnten zeigen, dass der Kontakt-Index sehr gut mit der Ausbreitung des Coronavirus in
             Deutschland korrelierte.
+            <a href="/images/example-diagram-contact-index.png" download="example-diagram-contact-index.png">
```

Comment:
Ich würde glaube den download wegnehmen und dafür nur den Link einsetzen, denke mal die meisten wollen es wenn dann zoomen und nicht herunterladen.

Reply:
Okay. Können wir gern so machen. Würdest du den Link in einem neuen Tab öffnen? Oder im selben Tab?

Reply:
im selben, dann kann man mit zurück einfach wieder aus dem zoom zurückspringen.

Reply:
Im selben Tab wuerde aber die App unmounten und dann wieder re-mounten?

Reply:
> Im selben Tab wuerde aber die App unmounten und dann wieder re-mounten?

Ja, das stimmt. 

Außerdem muss man den Back-Button vom Browser oder vom Smartphone nutzen und es gibt keine In-App-Möglichkeit. Das finde ich persönlich von der Benutzerführung nicht so toll.